features/bootstrap/FeatureContext.php

```diff
@@ -89,15 +92,44 @@ public function beforeScenario( $event ) {
 	 * @AfterScenario
 	 */
 	public function afterScenario( $event ) {
-		if ( !isset( $this->variables['RUN_DIR'] ) )
-			return;
+		if ( isset( $this->variables['RUN_DIR'] ) ) {
+			// remove altered WP install, unless there's an error
+			if ( $event->getResult() < 4 ) {
+				$this->proc( Utils\esc_cmd( 'rm -r %s', $this->variables['RUN_DIR'] ) )->run();
+			}
+		}
 
-		// remove altered WP install, unless there's an error
-		if ( $event->getResult() < 4 ) {
-			Process::create( Utils\esc_cmd( 'rm -r %s', $this->variables['RUN_DIR'] ), null, self::get_process_env_variables() )->run();
+		foreach ( $this->running_procs as $proc ) {
+			self::terminate_proc( $proc );
 		}
 	}
 
+	/**
+	 * Terminate a process and any of its children.
+	 */
+	private static function terminate_proc( $proc ) {
```

Comment:
The worst that could happen is you get a `network port already in use` error, which can be solved with `killall php`.